test/tests/functional/pbs_offline_vnodes.py

```diff
@@ -104,6 +106,46 @@ def create_multi_vnodes(self, num_moms, num_vnode=3):
             self.server.expect(NODE, {ATTR_NODE_state: 'free'},
                                id=self.moms.values()[i].shortname)
 
+    def verify_vnodes_state(self, expected_state):

```

Comment:
Change this to "verify_other_vnodes_state" instead of "verify_vnodes_state".

Reply:
That's what it used to be.  Arun asked for it to be changed.  The new name is verify_vnodes_state.  Is there some problem with the current name?

Reply:
I should have said that verify_vnodes_state() is not being called anywhere.

Reply:
Ahh.  Thank you!  It is now being called.  :)